src/style/result.css

```diff
@@ -0,0 +1,95 @@
+@import url('https://fonts.googleapis.com/css2?family=Chakra+Petch:wght@300&family=Neucha&display=swap');
+
+.result{
+    display: flex;
+    justify-content: center;
+    height: 100vh;
+    font-family: 'Chakra Petch', sans-serif;
+}
+
+.result__inner-container{
+    background-color: #1E1E1E;
+    width: 50%;
+    flex-direction: column;
+    padding-top: 30px;
+}
+
+.result__avatar-container{
+    background-color: #E1E1DA;
+    height: 200px;
+    width: 200px;
+    margin: auto;
+    border: #FFADCB solid 4px;
+}
+
+.result__text {
+    height: 50px;
+    padding: 10px;
+    font-size: 30px;
+    color: #E1E1DA;
+    font-weight: bold;
+}
+
+.result__question-text {
+    background-color: #E1E1DA;
+    height: 130px;
+    width: 70%;
+    padding: 10px;
+    font-size: 25px;
+    margin: auto;
+    border: #FFADCB solid 2px;
+}
+
+.result__answer-text {
+    color: #FFADCB;
+    font-size: 25px;
+    margin-bottom: 10px;
+}
+
+.result__user-answer {
+    margin-top: 25px;
+}
+
+.result__box {
+    height: 180px;
+    width: 90%;
+    margin: auto;
+    padding: 10px;
+    margin-top: 50px;
+}
+
+
+.result__title {
+    display: flex;
+    justify-content: space-between;
+    padding: 5px;
+}
+
+.result__slider {
+    height: 40px;
+    margin: auto;
+    display: flex;
+    flex-direction: row;
+}
+
+.result__first-answer {
+    background-color: #FFADCB;
+    height: 40px;
+    font-size: 20px;
+    font-weight: bold;
+}
+
+.result__second-answer {
+    background-color: #E1E1DA;
+    height: 40px;
```

Comment:
not sure I agree that all these heights being fixed in px is the best responsive option but we can look at this later, it's really not important